lib/facter/puppet_environmentpath.rb

```diff
@@ -0,0 +1,21 @@
+# This facter fact returns the value of the Puppet environmentpath. It is adapted
+# from the puppetlabs-stdlib puppet_vardir fact.
```

Comment:
Yes working on a PR now, however, puppet 4 needs to ship with Katello 3.2, and we branch next week.  

Also, there's no environment path on clients, and the kind of masterless setup the installer runs in is fairly unique.  I believe they rejected the foreman_cache_data change for similar reasons.